specs/experimental/admin-and-auth.md

```diff
@@ -25,21 +28,131 @@
+   threshold should also be modified to ensure it is the lowest value which is greater than 75% of
+   the number of owners. This is intended to meet Stage 1 requirements.
+
+1. **[The Foundation Upgrade
+   Safe](https://etherscan.io/address/0x847B5c174615B1B7fDF770882256e2D3E95b9D92):** This Safe is
+   one of the two owners of the ProxyAdminOwner Safe. It is also able to update the recommended and
+   required versions on the `ProtocolVersions` contract, given that observing the state of this
+   contract is optional, this is not considered to be affect safety and can therefore be managed the
+   Foundation Safe.
+
+   This Safe has a threshold of 5 and has 7 owners.
+
+1. **[The Foundation Operations
+   Safe](https://etherscan.io/address/0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A):** This Safe acts
+   as the Deputy Guardian, meaning that (via the Guardian Safes's `DeputyGuardianModule`) can call
+   any functions in the system which impact liveness.
+
+   This Safe has a threshold of 5 and has the same 7 owners as the Foundation Upgrade Safe.
+
+## Ownership model diagram
+
+The following diagram outlines the control relationships between the contracts in the
+
+```mermaid
+flowchart LR
+   subgraph SuperchainSystem[Superchain System]
+      subgraph Liveness
+         Pausability
+         OtherLiveness[Other Liveness controls]
+      end
+      subgraph Safety
+         ContractUpgrades
+         DisputeGame
+         OtherSafety[Other Safety]
+      end
+      subgraph NonProtocol[Out of Protocol]
+         PV[ProtocolVersions]
+      end
+   end
+
+
+   subgraph UpgradeSystem[Upgrade System]
+      FndUp[Foundation Upgrade Safe]
+      POA[ProxyAdminOwner Safe]
+      subgraph Security Council Safe
+         Council[Security Council Safe\n+ LivenessGuard]
+         LM[Liveness Module]
+      end
+   end
+
+   subgraph GuardianSystem[Guardian System]
+      FndOps[Foundation Ops Safe]
+      subgraph GuardianSafe[Guardian Safe]
+         GS[Guardian Safe]
+         DGM[Deputy Guardian Module]
+      end
+   end
+
+    POA -->|controls| Safety
+    FndUp --> POA
+    Council --> POA
+    Council --> GS
+    FndOps -->|pause\nunpause\nsetRespectedGameType\nblackListDisputeGame| DGM
+    FndUp -->|set versions|PV
+    LM -->|execTransactionFromModule| Council
+    DGM -->|execTransactionFromModule| GS
+    GS -->|controls| Liveness
+
+   %% Declare a class to make the outer boxes somewhat transparent to provide some contrast
+   classDef outer fill:#ffffff44
+   class SuperchainSystem,GuardianSystem,UpgradeSystem outer
+```
+
+Note: in the diagram above, the [`ProtocolVersions`
+   contract](../protocol/superchain-upgrades.md#protocolversions-l1-contract) is listed as "Out of
+   Protocol", because the decision to follow the version signals in the contract is optional. It is
+   included here for completeness, but is not considered as either Safety or Liveness affecting.
 
 ## Deputy guardian module
```

Comment:
Note to reviewers regarding the diff below this line: 

1. I ran an in IDE 'rewrap' which resulted in some whitespace changes below, but they should be superficial only. 
2. Some of the content below is outdated, I plan to address this in a follow up PR.

Reply:
Got it, so I will not review below this line